src/test/ui/const-ptr/forbidden_slices.32bit.stderr

```diff
@@ -243,7 +243,7 @@ error[E0080]: could not evaluate static initializer
 LL |         unsafe { intrinsics::ptr_offset_from_unsigned(self, origin) }
    |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |                  |
-   |                  ptr_offset_from_unsigned cannot compute offset of pointers into different allocations.
+   |                  ptr_offset_from_unsigned called on pointers into different allocations
```

Comment:
Interestingly the error does not even say that this is UB. It just says "could not evaluate static initializer". Maybe as part of https://github.com/rust-lang/miri/issues/2200 we should also improve the const-eval errors.